src/applications/disability-benefits/2346/containers/OrderTable.jsx

```diff
@@ -0,0 +1,186 @@
+import ErrorableCheckbox from '@department-of-veterans-affairs/formation-react/ErrorableCheckbox';
+import React from 'react';
+import { connect } from 'react-redux';
+import { focusElement } from '../../../../platform/utilities/ui';
+import { getDLCData, updateDLCData } from '../actions';
+
+class OrderHistory extends React.Component {
+  // NOTE: Using React state for checkboxes -@maharielrosario at 1/8/2020, 7:47:21 PM
+  // When discussed with other devs, the concensus was to have a stateful
+  // component to house our checkboxes' state. But all other state is in Redux.
+  // Is there an easy way to house individual checkbox state in Redux?
+
+  constructor(props) {
+    super(props);
+    this.createDlcUIData = this.createDlcUIData.bind(this);
```

Comment:
You don't need to bind `this` to `createDlcUIData` if you make `createDlcUIData` an arrow function 🙂

Reply:
@kelsonic OrderTable.jsx is also part of the 3614 branch work, so will update it there.